### PR TITLE
Implement database

### PR DESCRIPTION
**Pull request by juengerj**
Implements the tables and relationships for the back-end database. Also includes methods to test user creation and authentication and a route to test contexts and imports are working correctly.

### CODE REVIEW

File: animal_adoption/models/db.py [hunk 1]
```diff
@@ -1,2 +1,162 @@
...
 
+
+adoption_relationship = db.Table(
+    'RelationshipTable',
+    db.Column('adopter_id', db.Integer, db.ForeignKey('AdoptersTable.id_adopters')),
+    db.Column('animal_id', db.Integer, db.ForeignKey('AnimalsTable.id_animals'))
+)
+
+
+class Users(db.Model):
```
Comment (akikgt) [inline animal_adoption/models/db.py:13]: I was just curious. Which is more common in Flask to use singular or plural for a class name? (User or Users)
I thought a singular form is common but I'm not familiar with Flask/SQL alchemy convention
Feel free to use what you are comfortable with

**Comment by akikgt** [comment]
Great job! Other than my comment above, LGTM!

**Review by Ryandalion — APPROVED** [review]
Looks great Jason!

File: animal_adoption/models/db.py (hunk 1 above)
Comment (juengerj) [reply animal_adoption/models/db.py:13]: That's a good question. I will check before merging.

**Pull request merged**